homeassistant/components/tplink/light.py

```diff
@@ -267,9 +267,9 @@ def hs_color(self) -> tuple[int, int] | None:
         return hue, saturation
 
     @property
-    def supported_color_modes(self) -> set[ColorMode | str] | None:
+    def supported_color_modes(self) -> set[ColorMode] | None:
```

Comment:
```suggestion
    def supported_color_modes(self) -> set[ColorMode]:
```
This seems to always return a set.

Reply:
Thanks - adjusted accordingly.